core/src/main/java/nu/yona/server/properties/BatchServiceProperties.java

```diff
@@ -8,6 +8,8 @@
 {
 	private String serviceUrl = "http://localhost:8083";
 
+	private String activityAggregationJobCronExpression = "0 0 3 * * *";
```

Comment:
According to [this](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronSequenceGenerator.html) and [this](https://docs.spring.io/spring/docs/current/spring-framework-reference/html/scheduling.html) Spring documentation, my expression causes the job to run every day at three in the morning. Standard CRON expressions do not work with Spring, I encountered that at the test...